src/status_im/contexts/chat/home/new_chat/view.cljs

```diff
@@ -1,4 +1,4 @@
-(ns status-im.contexts.chat.new-chat.view
+(ns status-im.contexts.chat.home.new-chat.view
```

Comment:
Is it only one file that needs to be under `home` ?

Reply:
Not sure exactly what you mean here?

Reply:
> chat/home - any folder which may be used in the home page

I only see one file (and its style) under `contexts.chat.home`

Reply:
Oh okay fine, these files weren't showing as changes